packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxisRendering.selectors.ts

```diff
@@ -135,21 +136,67 @@ const selectorChartZoomAxisFilters = createSelector(
   },
 );
 
+export const selectorChartComputedXAxes = createSelector(
+  [
+    selectorChartRawXAxis,
+    selectorChartSeriesProcessed,
+    selectorChartSeriesConfig,
+    selectorChartZoomOptionsLookup,
+    selectorChartZoomAxisFilters,
+    (_, params: { drawingArea: ChartDrawingArea; zoomMap: Map<AxisId, ZoomData> | undefined }) =>
+      params,
+  ],
+  (axis, formattedSeries, seriesConfig, zoomOptions, getFilters, { drawingArea, zoomMap }) =>
```

Comment:
```suggestion
    (_, drawingArea: ChartDrawingArea) => drawingArea,
    (_, __, zoomMap: Map<AxisId, ZoomData> | undefined ) => zoomMap,
  ],
  (axis, formattedSeries, seriesConfig, zoomOptions, getFilters, drawingArea, zoomMap) =>
```

This with the following modification, and a memorization of `drawingArea` should avoid re-rendering the review axes

```diff
-  const { axis: xAxis, axisIds: xAxisIds } = useSelector(store, selectorChartComputedXAxes, {
-     drawingArea,
-     zoomMap: undefined,
-  });
+  const { axis: xAxis, axisIds: xAxisIds } = useSelector(store, selectorChartComputedXAxes, drawingArea, undefined);
```

Since the zoom preview should not be zoomed itself, the are the `zoomOptions`, and `getFilters` needed?